test/lambda-powertools-python-layer.test.ts

```diff
@@ -73,34 +87,34 @@ describe('with version configuration the construct', () => {
     });
 
     Template.fromStack(stack).hasResourceProperties('AWS::Lambda::LayerVersion', {
-      Description: 'Lambda Powertools for Python with Pydantic version 1.22.0',
+      Description: 'Lambda Powertools for Python with Extras version 1.22.0',
```

Comment:
np: we can be more explicit and say this Layer has **extra dependencies** or **additional dependencies**.